ui/store/actions.test.js

```diff
@@ -1485,23 +1485,20 @@ describe('Actions', () => {
 
     it('calls setUseBlockie in background', async () => {
       const store = mockStore();
-
-      const setUseBlockStub = background.setUseBlockie.callsFake((_, cb) =>
```

Comment:
These changes were needed because the tests referenced old wrapper functions, which were removed.

The fixtures here are still callback-based because the extension API used for cross-process communication is still callback-based.